configs/manifest.rb

```diff
@@ -0,0 +1,9 @@
+render "docker-compose.yaml.ctmpl"
+render "gpalloc.conf.ctmpl"
+copy_file "site.conf"
+
+copy_secret_from_path "secret/dsde/dsp-techops/common/server.crt"
+copy_secret_from_path "secret/dsde/dsp-techops/common/server.key"
+copy_secret_from_path "secret/dsde/dsp-techops/common/ca-bundle.crt"
+
+copy_secret_from_path "secret/dsde/firecloud/#{$env}/common/billing-account.pem", field = "private_key", output_file_name = "billing-account.pem"
```

Comment:
what populates `$env` here?

Reply:
ah. setting ENV in the environment. which we will need to do in deploy.sh but does it matter which billing pem we use? can we use the one in dev? do we always want to use the one in dev?

Reply:
I think test.firecloud.org and quality.firecloud.org have different master billing accounts, no?

Reply:
yeah. so i guess we do need to pass in an environment? where do you set that tho?